include/picongpu/fields/absorber/Absorber.hpp

```diff
@@ -178,6 +182,116 @@ namespace detail
         { Absorber::zNegativeNumCells, Absorber::zPositiveNumCells }
     };
 
+    //! Thickness of the absorbing layer
```

Comment:
Could you please add a description that this object should not be stored over multiple timesteps. Since communication maks are used the cached values will be only valid within one simulation step.

Reply:
This concerns not the object itself, but whether `getLocalThickness()` results can be reused on another time step (which they can't generally). I will add an explanatory note there.